lib/action_cable_client/version.rb

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 class ActionCableClient
-  VERSION = '1.3.4'
+  VERSION = '1.3.5'
```

Comment:
I'll handle version bump on release. looks like because I don't have the simplecov / codeclimate gem versions locked, I need to deal with an upgrade issue with those.